iidm/iidm-api/src/main/java/com/powsybl/iidm/network/TerminalRef.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network;
+
+import com.powsybl.commons.PowsyblException;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public final class TerminalRef {
+
+    public enum Side {
+        ONE,
+        TWO,
+        THREE
+    }
+
+    private final String id;
+    private final Side side;
```

Comment:
The `Side` should be an optional, no?

Reply:
We always define it. When there is an injection we consider `Side.ONE` but it can be also managed as optional. I will change it